spacy/util.py

```diff
@@ -1279,6 +1297,25 @@ def dot_to_object(config: Config, section: str):
     return component
 
 
+def set_dot_to_object(config: Config, section: str, value: Any) -> None:
+    """Update a config at a given position from a dot notation.
+
+    config (Config): The config.
+    section (str): The dot notation of the section in the config.
+    value (Any): The value to set in the config.
+    """
+    component = config
+    parts = section.split(".")
+    for i, item in enumerate(parts):
+        try:
```

Comment:
Is there no alternative to this try/except?

Reply:
🤷‍♀️ It's also how we do it in the very similar `dot_to_object` helper. I guess the alternative would be to do an instance check for dict and check if the key exists. Not sure if there are any other cases we'd have to cover.